packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanBarRow.js

```diff
@@ -48,6 +49,7 @@ type SpanBarRowProps = {
   },
   serviceName: string,
   showErrorIcon: boolean,
+  span: Span,
```

Comment:
Now that the `Span` is being passed in as a prop, I think the following props can be removed, with the values derived in `render()`

- `depth`
- `label`
- `isParent`
- `operationName`
- `serviceName`
- `spanID`

Reply:
just applied this, it helped clean up a couple of files 👍

Reply:
Awesome